webapp/channels/src/components/create_user_groups_modal/create_user_groups_modal.tsx

```diff
@@ -278,31 +297,32 @@ export class CreateUserGroupsModal extends React.PureComponent<Props, State> {
                                 value={this.state.mention}
                                 maxLength={64}
                                 data-testid='mentionInput'
+                                required={true}
                                 customMessage={{type: ItemStatus.ERROR, value: this.state.mentionInputErrorText}}
                             />
                         </div>
-                        <h2>
-                            <FormattedMessage
-                                id='user_groups_modal.addPeople'
-                                defaultMessage='Add People'
-                            />
-                        </h2>
-                        <div className='group-add-user'>
-                            <AddUserToGroupMultiSelect
-                                multilSelectKey={'addUsersToGroupKey'}
-                                onSubmitCallback={this.createGroup}
-                                focusOnLoad={false}
-                                savingEnabled={this.isSaveEnabled()}
-                                addUserCallback={this.addUserCallback}
-                                deleteUserCallback={this.deleteUserCallback}
-                                backButtonText={defineMessage({id: 'multiselect.cancelButton', defaultMessage: 'Cancel'})}
-                                backButtonClick={
-                                    typeof this.props.backButtonCallback === 'function' ? this.goBack : this.doHide
-                                }
-                                backButtonClass={'multiselect-back'}
-                                saving={this.state.saving}
-                            />
-                        </div>
+                        <fieldset className='group-add-people-fieldset'>
```

Comment:
Curious why fieldset is proper here and not in the input component for the input and the label we had before the changes below?

Reply:
Was a recommendation on this ticket: https://mattermost.atlassian.net/browse/MM-64019